src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -310,6 +310,7 @@ Apply to thead to make headers not bold.
 
 .text-preserve-space {
     white-space: pre-wrap;
+    word-break: break-all;
```

Comment:
Yes, but depends on the inherited/given width of the element. So words break take less priority over protecting the element width. 

However, to avoid any doubts, Let's make a different class and apply only for feedback column tds then.